app/controllers/submit.js

```diff
@@ -556,22 +604,51 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             let currentAbstract = node.get('description');
             let currentTags = node.get('tags').slice(0);
             let currentDOI = model.get('doi');
+            let currentLicenseType = model.get('license');
+            let currentLicenseRecord = model.get('licenseRecord');
+            let currentNodeLicenseType = node.get('license');
+            let currentNodeLicenseRecord = node.get('nodeLicense');
+            let newCopyrightHolders = [''];
+            if (this.get('basicsLicense.copyrightHolders') && this.get('basicsLicense.copyrightHolders').length) {
+                newCopyrightHolders = this.get('basicsLicense.copyrightHolders').split(',');
+            }
 
             if (this.get('abstractChanged')) node.set('description', this.get('basicsAbstract'));
             if (this.get('tagsChanged')) node.set('tags', this.get('basicsTags'));
+            if (this.get('licenseChanged') && this.get('applyLicense')) {
```

Comment:
`licenseChanged` is often true, when I have changed nothing. For example, I save my license, go to authors, and then return to basics and add a tag. I haven't touched the license. licenseChanged is still true, and then you end up unnecessarily putting the license stuff in the PATCH request.  

There was some discussion when I implemented this page over only sending in the relevant fields and worried about the small risk of collisions, hence the checking to make sure if a field has changed before setting it.